src/composables/translateParser.ts

```diff
@@ -0,0 +1,30 @@
+import { TOptions } from 'i18next'
+import { useTranslation } from 'i18next-vue'
+
+/**
+ * Translate legacy _V3 text
+ * eg: from getLegendHtml, server returns html to be translated with angular directive "<h4 translate>Rigoles d&#39;érosion identifiées</h4>"
+ * @returns
+ */
+export function useTranslateParser() {
```

Comment:
translation parser would be rather a service than a composable, no reactivity involved.

Reply:
It is a composable because it uses i18next's `useTranslation().`